### PR TITLE
sd-network: DHCPv6 - Add status codes

### PR DESCRIPTION
**Pull request by ssahani**
https://www.iana.org/assignments/dhcpv6-parameters/dhcpv6-parameters.xhtml#dhcpv6-parameters-5

### CODE REVIEW

File: src/libsystemd-network/dhcp6-protocol.h [hunk 1]
```diff
@@ -83,14 +83,30 @@ enum {
 };
 
 enum {
```
Comment (poettering) [inline src/libsystemd-network/dhcp6-protocol.h:89]: can you add a comment here with the URL from the commit msg, too?

File: src/libsystemd-network/dhcp6-protocol.h [hunk 2]
```diff
@@ -83,14 +83,30 @@ enum {
...
+        DHCP6_STATUS_UNKNOWN_QUERY_TYPE           = 7,
+        DHCP6_STATUS_MALFORMED_QUERY              = 8,
+        DHCP6_STATUS_NOT_CONFIGURED               = 9,
+        DHCP6_STATUS_NOT_ALLOWED                  = 10,
+        DHCP6_STATUS_QUERY_TERMINATED             = 11,
+        DHCP6_STATUS_DATA_MISSING                 = 12,
+        DHCP6_STATUS_CATCHUP_COMPLETE             = 13,
+        DHCP6_STATUS_NOT_SUPPORTED                = 14,
+        DHCP6_STATUS_TLS_CONNECTION_REFUSED       = 15,
+        DHCP6_STATUS_ADDRESSIN_USE                = 16,
```
Comment (poettering) [inline src/libsystemd-network/dhcp6-protocol.h]: …ADDRESSIN_USE → …ADDRESS_IN_USE

File: src/libsystemd-network/dhcp6-protocol.h [hunk 3]
```diff
@@ -83,14 +83,30 @@ enum {
...
+        DHCP6_STATUS_CONFIGURATION_CONFLICT       = 17,
+        DHCP6_STATUS_MISSING_BINDING_INFORMATION  = 18,
+        DHCP6_STATUS_OUTDATED_BINDING_INFORMATION = 19,
+        DHCP6_STATUS_SERVER_SHUTTING_DOWN         = 20,
+        DHCP6_STATUS_DNS_UPDATE_NOT_SUPPORTED     = 21,
+        DHCP6_STATUS_EXCESSIVE_TIMESKEW           = 22,
```
Comment (poettering) [inline src/libsystemd-network/dhcp6-protocol.h]: …_TIMESKEW → …_TIME_SKEW

**Comment by ssahani** [comment]
Updated  as suggested. PTAL. Thanks for the review.

**Pull request merged**